### ISSUE TITLE
Install-WindowsUpdate never install last update automatically

### ISSUE THREAD

**Issue by tetatetit** [chocolatey/boxstarter#344]
At least on 1709 and 1803 pro/enterprise right after clean installation on this simple test https://raw.githubusercontent.com/tetatetit/mybox/master/test.ps1 (the key nuance is "Notify for download and auto install" AUOptions in Windows Update policy which can be set via gpedit.msc "Computer Configuration\Administrative Templates\Windows Components\Windows Update\Configure Automatic Updates"): Result - last update is ever impossible to install via Install-WindowsUpdate but infinite reboot loop instead

**Comment by pauby** [comment]
@tetatetit Appreciate this was some time ago but did you ever resolve this?

**Comment by tetatetit** [comment]
@pauby Yes, I have resolved this issue in my fork and tested already in production that it works well so I have made pull request #317 to merge it with this mainstream repo, but my pull request has not been accepted yet

**Comment by pauby** [comment]
@tetatetit Thanks for doing that. Appreciate it's been a whiel since you raised that PR. We've been quiet for some time but we're going through the issues and pull requests to categorise (which has largely been done) and progress them. Bear with us and we'll get this sorted. 

I've amended the PR title to make it clear it's linked to this issue.
